### ISSUE TITLE
Build Web/Mobile UI Clients for the backend services students will build

### ISSUE THREAD

**Issue by suyashkumar** [mlp6/Medical-Software-Design#14]
[Nice to have] Build sample clients to consume the backend APIs students will be building in class.

**Comment by mlp6** [comment]
Would this be equivalent to some of the functionality you said Matlab could provide with direct data access from the server?

**Comment by suyashkumar** [comment]
Indeed! Students in class may be required to use Matlab's [webread](https://www.mathworks.com/help/matlab/ref/webread.html) command to interact with and pull data from the server.

**Comment by suyashkumar** [comment]
`webread` is pretty nifty--it will automatically parse JSON, XML, etc. into native Matlab structures.
